docs/monitoring/metrics.md

```diff
@@ -856,66 +918,80 @@ Thus, in order to infer the metric identifier:
 <table class="table table-bordered">
   <thead>
     <tr>
-      <th class="text-left" style="width: 20%">Scope</th>
-      <th class="text-left" style="width: 30%">Metrics</th>
-      <th class="text-left" style="width: 50%">Description</th>
+      <th class="text-left" style="width: 18%">Scope</th>
+      <th class="text-left" style="width: 26%">Metrics</th>
+      <th class="text-left" style="width: 48%">Description</th>
+      <th class="text-left" style="width: 8%">Type</th>
     </tr>
   </thead>
   <tbody>
     <tr>
       <th rowspan="7"><strong>Task</strong></th>
       <td>currentLowWatermark</td>
-      <td>The lowest watermark this task has received.</td>
+      <td>The lowest watermark this task has received (in milliseconds).</td>
+      <td>Gauge</td>
     </tr>
     <tr>
       <td>numBytesInLocal</td>
       <td>The total number of bytes this task has read from a local source.</td>
+      <td>Counter</td>
```

Comment:
these metrics are all missing units. (bytes, records, and in particular latency (milliseconds)

Reply:
Yes.
But, eg. 
**numBytesInRemote** The total number of bytes this task has read from a remote source.
**numBytesInRemotePerSecond**  The number of bytes this task reads from a remote source per second.

Users can easily know that these metrics units through this description, will not be confused.

So, I think we do not need to add additional unit descriptions to these metrics (except latency metric).